cpp/src/Ice/SSL/SchannelEngine.cpp

```diff
@@ -1344,13 +1335,13 @@ Schannel::SSLEngine::createServerAuthenticationOptions() const
                     CertDuplicateCertificateContext(cert);
                 }
 
-                return SCHANNEL_CRED{
-                    .dwVersion = SCHANNEL_CRED_VERSION,
+                return SCH_CREDENTIALS{
+                    .dwVersion = SCH_CREDENTIALS_VERSION,
                     .cCreds = static_cast<DWORD>(_allCerts.size()),
                     .paCred = const_cast<PCCERT_CONTEXT*>(_allCerts.size() > 0 ? &_allCerts[0] : nullptr),
                     // Don't set SCH_SEND_ROOT_CERT as it seems to cause problems with Java certificate validation and
                     // Schannel doesn't seems to send the root certificate either way.
-                    .dwFlags = SCH_CRED_NO_SYSTEM_MAPPER | SCH_CRED_DISABLE_RECONNECTS | SCH_USE_STRONG_CRYPTO};
+                    .dwFlags = SCH_CRED_NO_SYSTEM_MAPPER | SCH_USE_STRONG_CRYPTO};
```

Comment:
We used to hardcode `SCH_CRED_DISABLE_RECONNECTS` in the default settings, but that breaks TLS 1.3 on Windows 11/Windows Server 2022.